src/test/java/uk/gov/hmcts/reform/slc/services/sendletterclient/UpdateSentToPrintAtTest.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.reform.slc.services;
+package uk.gov.hmcts.reform.slc.services.sendletterclient;
```

Comment:
How about just `client`?

Reply:
I named the package the same as tested service (I'll add a new test class for `updatePrintedAt` method to this package soon.